### PR TITLE
Proofreading dev documents under /user_doc/guide/task

### PR DESCRIPTION
**Pull request by Tianqi-Dotes**
Proofreading dev documents under /user_doc/guide/task

### CODE REVIEW

File: docs/en-us/dev/user_doc/guide/task/conditions.md [hunk 1]
```diff
@@ -17,20 +17,20 @@ Drag in the toolbar<img src="/img/conditions.png" width="20"/>The task node to t
...
+- Run flag: Identifies whether this node schedules normally, if it does not need to execute, select the `prohibition execution`.
+- Descriptive information: Describe the function of the node.
+- Task priority: When the number of worker threads is insufficient, execute in the order of priority from high to low, and tasks with the same priority will execute in a first-in first-out order.
+- Worker grouping: Assign tasks to the machines of the worker group to execute. If `Default` is selected, randomly select a worker machine for execution.
+- Times of failed retry attempts: The number of times the task failed to resubmit. You can select from drop-down or fill-in a number.
+- Failed retry interval: The time interval for resubmitting the task after a failed task. You can select from drop-down or fill-in a number.
+- Timeout alarm: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
+- Downstream tasks selection: supports two branches: success and failure
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/task/conditions.md:27]: I found we use two `:` in one line, it is ok or a mistake?

File: docs/en-us/dev/user_doc/guide/task/map-reduce.md [hunk 2]
```diff
@@ -1,73 +1,73 @@
...
+-    **Node name**: The node name in a workflow definition is unique.
+-    **Run flag**: Identifies whether this node schedules normally, if it does not need to execute, select the `prohibition execution`.
+-    **Descriptive information**: Describe the function of the node.
+-    **Task priority**: When the number of worker threads is insufficient, execute in the order of priority from high to low, and tasks with the same priority will execute in a first-in first-out order.
+-    **Worker grouping**:  Assign tasks to the machines of the worker group to execute. If `Default` is selected, randomly select a worker machine for execution.
+-    **Environment Name**: Configure the environment name in which run the script.
+-    **Times of failed retry attempts**: The number of times the task failed to resubmit.
+-    **Failed retry interval**: The time interval (unit minute) for resubmitting the task after a failed task.
+-    **Delayed execution time**: The time (unit minute) that a task delays in execution.
+-    **Timeout alarm**: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/task/map-reduce.md:23]: Should we remove the unnessary space here?
```suggestion
- **Timeout alarm**: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
```

File: docs/en-us/dev/user_doc/guide/task/conditions.md (hunk 1 above)
Comment (Tianqi-Dotes) [reply docs/en-us/dev/user_doc/guide/task/conditions.md:27]: done changes

File: docs/en-us/dev/user_doc/guide/task/map-reduce.md (hunk 2 above)
Comment (Tianqi-Dotes) [reply docs/en-us/dev/user_doc/guide/task/map-reduce.md:23]: done changes

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Comment by zhongjiajie** [comment]
Thanks~

**Pull request merged**